diff --git a/aeneas/ttswrappers/macosttswrapper.py b/aeneas/ttswrappers/macosttswrapper.py
index cb85ed608a90f0528cc8558b1298d9e238defc47..e2137436f5395c7639b206b3f9b278f5f3e7c959 100644
--- a/aeneas/ttswrappers/macosttswrapper.py
+++ b/aeneas/ttswrappers/macosttswrapper.py
@@ -218,7 +218,7 @@ class MacOSTTSWrapper(BaseTTSWrapper):
     }
     DEFAULT_LANGUAGE = ENG
 
-    OUTPUT_AUDIO_FORMAT = ("pcm_s16le", 1, 22050)
+    OUTPUT_AUDIO_FORMAT = ("pcm_s16le", 1, 16000)
 
     HAS_SUBPROCESS_CALL = True
 
@@ -235,5 +235,5 @@ def __init__(self, rconf=None, logger=None):
             self.CLI_PARAMETER_WAVE_PATH,           # it will be replaced by the actual output file
             self.CLI_PARAMETER_TEXT_STDIN,          # text is read from stdin,
             u"--data-format",                       # set output data format
-            u"LEF32@22050"                          # data format string
+            u"LEI16@16000"                          # data format string
         ])
